frontend/src/components/mine/Incidents/MineIncidents.js

```diff
@@ -59,10 +59,18 @@ export class MineIncidents extends Component {
   }
 
   handleAddMineIncident = (values) => {
-    this.props.createMineIncident(this.props.mine.mine_guid, values).then(() => {
-      this.props.closeModal();
-      this.props.fetchMineIncidents(this.props.mine.mine_guid);
-    });
+    // set default fatalities & injuries to 0 if not set by user
```

Comment:
I think this line of code is self-documenting, and the comment isn't needed 🙂

Reply:
removed.